src/liquid/components/ld-notice/ld-notice.tsx

```diff
@@ -8,21 +9,41 @@ import { Component, h, Prop, Host } from '@stencil/core'
  * @part icon - Image tag used for the icon
  */
 @Component({
+  assetsDirs: ['assets'],
```

Comment:
```suggestion
```

Reply:
isn't this the way to point to the assets directory?

Reply:
I just haven't seen that you are using any asset. You should only need it if you are using the [`getassetpath`](https://stenciljs.com/docs/assets#getassetpath) function.